components/gripper/gripper.go

```diff
@@ -79,6 +79,12 @@ func FromRobot(r robot.Robot, name string) (Gripper, error) {
 	return robot.ResourceFromRobot[Gripper](r, Named(name))
 }
 
+// FromDependencies is a helper for getting the named sensor from a collection of
+// dependencies.
+func FromDependencies(deps resource.Dependencies, name string) (Gripper, error) {
+	return resource.FromDependencies[Gripper](deps, Named(name))
+}
+
 // NamesFromRobot is a helper for getting all gripper names from the given Robot.
 func NamesFromRobot(r robot.Robot) []string {
```

Comment:
ngl didn't know this was a method on some of the apis either

Reply:
makes sense - none of the services have this so it's a components-only helper ™️

Reply:
plus you joined components after we added all this noise.